docs/accessing-services.asciidoc

```diff
@@ -94,7 +94,7 @@ This section only covers TLS certificates for the HTTP layer. Those for the tran
 ==== Default self-signed certificate
 
 By default, the operator manages a self-signed certificate with a custom CA for Elasticsearch, Kibana and APM Server.
-The CA, the certificate and the private key are each stored in a `Secret`.
+The CA, the certificate and the private key are stored in a `secret`.
```

Comment:
I think the intent behind `each` here was to point out that we are dealing with three different secrets one for the CA one for private key (and technically also the certificate) and one for the certificate

Reply:
It sounds a bit weird, but I can revert back to "each"

Reply:
"The CA, the certificate and the private key are **respectively** stored in a `Secret`."
Is respectively better than each?

Or:
"The CA, the certificate and the private key are **respectively** stored in three different `Secrets`."

Reply:
"Respectively" sounds OK. Thanks Thibault! 
Question: Secret is capital or lower case?

Reply:
> Question: Secret is capital or lower case?

Now I tend to write it with a capital.
I don't think we have already decided a rule to follow when we use the name of a Kubernetes API object.
As suggested in #1252, using camel case for k8s API objects is a best practice in the K8S documentation.
Maybe it will be more efficient to track this in a dedicated issue.

Reply:
To bike shed a little bit here, I think "each stored in a separate secret" makes it clear what's going on

Reply:
I'll go for that, thank you @anyasabo!